cli/compose/loader/loader_multiple_test.go

```diff
@@ -0,0 +1,114 @@
+package loader
+
+import (
+	"testing"
+
+	"github.com/docker/cli/cli/compose/types"
+	"github.com/stretchr/testify/require"
+)
+
+/*
+func TestLoadTwoDifferentVersion(t *testing.T) {
+	configDetails := types.ConfigDetails{
+		ConfigFiles: []types.ConfigFile{
+			{Filename: "base.yml", Config: map[string]interface{}{
+				"version": "2.0",
+			}},
+			{Filename: "override.yml", Config: map[string]interface{}{
+				"version": "3.0",
+			}},
+		},
+	}
+	_, err := Load(configDetails)
+	require.Error(t, err)
+}
+*/
+
+func TestSimpleLoad(t *testing.T) {
+	base := map[string]interface{}{
+		"version": "3.4",
+		"services": map[string]interface{}{
+			"foo": map[string]interface{}{
+				"image": "foo",
+				"build": map[string]interface{}{
+					"context":    ".",
+					"dockerfile": "bar.Dockerfile",
+				},
+				"labels": []interface{}{
+					"foo=bar",
+				},
+				"cap_add": []interface{}{
+					"NET_ADMIN",
+				},
+			},
+		},
+		"volumes":  map[string]interface{}{},
+		"networks": map[string]interface{}{},
+		"secrets":  map[string]interface{}{},
+		"configs":  map[string]interface{}{},
+	}
```

Comment:
mmmm can we extract something ? :)

Reply:
I'm going to write a builder for that once it's in `code-review` yes 😉